### ISSUE TITLE
TypeError: Invalid badgeNumber specified.

### ISSUE THREAD

**Issue by LordTrips** [KK-Designs/UpdateHub#78]
```
TypeError: Invalid badgeNumber specified.
Expected: number
Got: object
    at Badge.update (C:\Users\C\AppData\Local\Programs\update-hub\resources\app.asar\node_modules\electron-taskbar-badge\lib\cjs\index.js:66:10)
    at SystemPreferences.<anonymous> (C:\Users\C\AppData\Local\Programs\update-hub\resources\app.asar\node_modules\electron-taskbar-badge\lib\cjs\index.js:50:9)
    at SystemPreferences.emit (node:events:513:28)
```

---

update-hub 2.0.0
Electron 25.3.0
win32 10.0.19045
Locale: en-US

**Comment by KK-Designs** [comment]
We are trying to fix this error. Thank you for reporting this

**Comment by KK-Designs** [comment]
Should be fixed in [v2.2.0](https://github.com/KK-Designs/UpdateHub/releases/tag/v2.2.0). Closing as completed
